### PR TITLE
feat(db): Allow both name and title parameters

### PR DESCRIPTION
**Pull request by SishaarRao**
**Why: **As we migrate to new column name, we want to still accept title as a valid parameter since we're in the middle of migration

**What: ** Accept both title and name as valid parameters in the POST request

Details:
- Add name as a possible field in the create POST request
- Change required field to having either name OR title as required, but not both
- Add tests for new validation rules

### CODE REVIEW

File: lib/validators/movie.js
Comment (Lianathanoj) [inline lib/validators/movie.js:6]: hm, so it looks like we don't allow `name` if `title` exists or vice versa, but what happens if we just pass in `title`? there may still be gaps if we're not double writing both the `title` and the `name` column during this step. that is, if we pass in `name`, the `title` value will not be written, meaning that customers would not see the desired output as there are still read dependencies on the `title` column. if we pass in `title`, then we're not writing to the `name` column, meaning there will be gaps when we backfill.

essentially, we want to double write the `name` and `title` columns if one of `title` or `name` are passed in. does that make sense?

**Review by Lianathanoj — APPROVED** [review]
LGTM! for reference, even though miyagi said to keep the API interface the same, this approach also works for a different use case for how we deprecate and use new naming schemes for request payloads. as such, allowing `name` or `title` in this case while keeping double writes is fine.

File: test/validators/movie.test.js
Comment (jekatlob) [inline test/validators/movie.test.js]: This is kind of misleading, since it will actually accept `name`.

**Review by jekatlob — CHANGES_REQUESTED** [review]
I'm not going to block the PR because of the change in API interface, but I will point out that as a result of that decision your test description is misleading (see the inline comment). I'll leave it to you as to how to reconcile it (I'm fine with either tweaking the API changes or the test description).

File: test/validators/movie.test.js
Comment (SishaarRao) [reply test/validators/movie.test.js]: Resolved: Changed test description

**Pull request merged**
